index.html

```diff
@@ -111,51 +112,74 @@ <h2>STAC Specification</h2>
 
 									<p><ul style="display: inline-block; text-align: left;"><li><b><a href="https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md">STAC Item</a></b> is the core atomic unit, representing a single spatiotemporal asset as a <a href="http://geojson.org">GeoJSON</a> feature plus datetime and links.</li>
 									<li><b><a href="https://github.com/radiantearth/stac-spec/blob/master/catalog-spec/catalog-spec.md">STAC Catalog</a></b> is a simple, flexible JSON file of links that provides a structure for STAC Items together to be crawled or browsed.</li>
-									<li><b><a href="https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md">STAC Collection</a></b> provides additional information about a collection of data, such as the extents, license, keywords, providers, etc.</li>
+									<li><b><a href="https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md">STAC Collection</a></b> is an extension of the STAC Catalog with additional information such as the extents, license, keywords, providers, etc.  STAC Items are grouped into STAC Collections at the lowest level.</li>
```

Comment:
I'm confused by "at the lowest level"... What does that mean?

Reply:
This is my poor language attempting to encapsulate what I think are the two core differentiations between a collection and a catalog:
1) Additional required info from a catalog
2) Items can't be added to catalogs that aren't collections - getting this from sat-utils/sat-stac requirements

Maybe a rephrase to:
STAC Items must be added to a Collection, rather than to a Catalog.

Obviously open to be corrected or seeing another way to describe this, but I think we need to make it clear up front why a Collection exists, and as far as I can tell it's to group items.

Reply:
Items can be added to Catalogs. Collections are used to describe datasets better.

You'd describe Sentinel 2A L1C as Collection, but subsets of it as catalogs. For example all data from 2018 could be in a catalog or so...

Reply:
Text updated accordingly.